packages/server/src/automations/steps/utils.ts

```diff
@@ -19,15 +19,15 @@ export async function getFetchResponse(fetched: any) {
 // need to make sure all ctx structures have the
 // throw added to them, so that controllers don't
 // throw a ctx.throw undefined when error occurs
-// opts can contain, body, params and version
+// opts can contain, body, params, version, and user
 export function buildCtx(
   appId: string,
   emitter?: ContextEmitter | null,
   opts: any = {}
 ) {
   const ctx: any = {
```

Comment:
Should we type these `any` to ensure that stuff like user is properly populated?

Reply:
It's _really_ hard to type this correct. `UserCtx` is a really complex object.